messenger.rst

```diff
@@ -762,7 +762,59 @@ message before terminating.
 
 However, you might prefer to use different POSIX signals for graceful shutdown.
 You can override default ones by setting the ``framework.messenger.stop_worker_on_signals``
-configuration option.
+configuration option:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/messenger.yaml
+        framework:
+            messenger:
+                stop_worker_on_signals:
+                    - SIGINT
```

Comment:
What is `SIGINT` in this option? A string? (`'SIGINT'`) A PHP constant? (`!php/const:SIGINT`) Other? Thanks.

Reply:
If the PR gets accepted as is, it will be strings. That's updated, thanks for pointing this out!